### ISSUE TITLE
References that have recommended storing local trees rather than ARG structures

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/what-is-an-arg-paper#119]
ISTR that in the ARGweaver docs, it recommends using the newick-based local trees format (I think they call it `.smc` rather than their `.arg` format, specifically because it is tedious to extract the local trees from their `.arg` format. This is exactly the sort of back-up we want to justify the edge-annotated encoding. It would be useful to dig up the quote (I can do that) and also find other examples where similar recommendations have been made, if they exist.

**Comment by jeromekelleher** [comment]
Unless it's in something citable, I'm not sure it'll be worth mentioning in the paper?

**Comment by hyanwong** [comment]
> Unless it's in something citable, I'm not sure it'll be worth mentioning in the paper?

Yes, that is true. Probably worth keeping this issue open in case there are citable references that say that the node-annotated Griffiths-style ARG format makes it hard to extract trees. It would be a good thing to find, as it would be strong independent justification for our approach.
